### PR TITLE
Update DG

### PR DESCRIPTION
**Pull request by SpecOps2016**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -335,6 +335,30 @@ Alternatives:
     - Pros: Easy to type
     - Cons: Easier to type the wrong short-form command as they differ by 1 letter
 
+
+
+### Sort Feature
+
+#### Current Implementation
+
+The sort feature allows the user to sort the list of patients, appointments and bills in the application. The sorting is done.
```
Comment (guokweijie) [inline docs/DeveloperGuide.md:344]: Don't need to mention the sorting is done.

**Review by guokweijie — COMMENTED** [review]
small fix required

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (SpecOps2016) [reply docs/DeveloperGuide.md]: fixed

**Review by Fuzanwenn — APPROVED** [review]
LGTM

**Review by guokweijie — APPROVED** [review]
LGTM

**Pull request merged**